android-agent/src/main/java/io/opentelemetry/android/internal/services/network/data/Carrier.java

```diff
@@ -12,15 +12,15 @@
 import java.util.Objects;
 
 @RequiresApi(api = Build.VERSION_CODES.P)
-final class Carrier {
+public final class Carrier {
```

Comment:
Add disclaimer about `internal` package.

Reply:
Might be a nice choice for a kotlin data class? Certainly can wait tho.

Reply:
Thanks, I've made the changes.